# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/rene/imlib2/configure.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- imlib2-2002-08-18/configure.in.orig	2002-12-31 01:09:29.000000000 +0100
+++ imlib2-2002-08-18/configure.in	2002-12-31 01:06:40.000000000 +0100
@@ -300,10 +300,6 @@
 AC_SUBST(x_libs)
 AC_SUBST(dlopen_libs)
 
-AC_OUTPUT(imlib2-config Makefile loaders/Makefile src/Makefile test/Makefile \
-          filters/Makefile demo/Makefile doc/Makefile \
-	  ,
-[
-test -z "$CONFIG_HEADERS" || echo timestamp > stamp-h;
-chmod +x imlib2-config
-])
+AC_CONFIG_FILES(imlib2-config Makefile loaders/Makefile src/Makefile \
+                test/Makefile filters/Makefile demo/Makefile doc/Makefile)
+AC_OUTPUT()
